kyo-core/shared/src/main/scala/kyo/randoms.scala

```diff
@@ -1,27 +1,108 @@
 package kyo
 
-abstract class Random:
+trait Random:
     def nextInt: Int < IOs
     def nextInt(n: Int): Int < IOs
     def nextLong: Long < IOs
     def nextDouble: Double < IOs
     def nextBoolean: Boolean < IOs
     def nextFloat: Float < IOs
     def nextGaussian: Double < IOs
+    def nextValue[T](seq: Seq[T]): T < IOs
+    def nextValues[T](length: Int, seq: Seq[T]): Seq[T] < IOs
+    def nextString(length: Int): String < IOs
+    def nextString(length: Int, chars: Seq[Char]): String < IOs
+    def nextBytes(length: Int): Seq[Byte] < IOs
```

Comment:
I've made this method return a `Seq` instead of `Array` since it's immutable and might prevent newcomers from making mistakes.